src/components/molecules/Message/Message.jsx

```diff
@@ -29,8 +33,10 @@ export const MessageComponent = ({ children, date, isYours, profileID }) => {
                     {children}
                 </Text>
             </div>
-        )
-    );
+        );
+    }
+
+    return null;
```

Comment:
returning null should be first, more clean code, I mean replace that with
if(!userDetails.data){
return null
}